rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -101,13 +99,13 @@ def test_paraswap_swap_with_fee(accountant: 'Accountant', db_settings: dict):
 
     accounting_history_process(
         accountant=accountant,
-        start_ts=TIMESTAMP_0_SEC,
-        end_ts=TIMESTAMP_1_SEC,
+        start_ts=TIMESTAMP_1700000000_SEC,
+        end_ts=TIMESTAMP_1700000001_SEC,
         history_list=events,
     )
     no_message_errors(accountant.msg_aggregator)
 
-    fee_spent_event = accountant.pots[0].processed_events[4]
+    fee_spent_event = accountant.pots[0].processed_events[3]
```

Comment:
Why did thus change here? The timing of events should not have changed at all.

Reply:
this is because earlier we were getting a "Prefork acquisition for ETC" processed event because the timestamp was < ETH_DAO_FORK_TS. Now it's not